functions/arrayLoop/1.0/index.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-await-in-loop */
+const loop = async ({ array = [], outputType }, steps) => {
+  for (let index = 0; index < array.length; index += 1) {
```

Comment:
Change to index++

Reply:
I use the same method as in the native loop function: https://github.com/bettyblocks/native-functions/blob/master/functions/loop/1.0/index.js @jwielinga